pkgs/by-name/ya/yafc-ce/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+  SDL2,
+  SDL2_image,
+  SDL2_ttf,
+}:
+buildDotnetModule rec {
+  pname = "yafc-ce";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "shpaass";
+    repo = pname;
```

```suggestion
    repo = "yafc-ce";
```
https://github.com/nix-community/nixpkgs-lint/issues/21